pkg/admission/apibinding/apibinding_admission.go

```diff
@@ -170,6 +184,17 @@ func (o *apiBindingAdmission) Validate(ctx context.Context, a admission.Attribut
 		return admission.NewForbidden(a, fmt.Errorf("workspace reference is missing")) // this should not happen due to validation
 	}
 
+	// Verify the labels
+	value, found := apiBinding.Labels[apisv1alpha1.InternalAPIBindingExportLabelKey]
+	if apiBinding.Spec.Reference.Workspace == nil && found {
+		return admission.NewForbidden(a, fmt.Errorf("metadata.labels.%s must not be set", apisv1alpha1.InternalAPIBindingExportLabelKey))
+	} else if expected := permissionclaims.ToAPIBindingExportLabelValue(
```

Comment:
won't this be sometimes set to `ToReflexiveAPIBindingLabelKeyAndValue`?

Reply:
no, there are two labels that matter here: 
1. `claimed.internal.apis.kcp.dev/hash: hash`. This defaults to the reflexive value for reflexive APIBindings (= of the same export). 
2. `ToAPIBindingExportLabelValue` is for the `internal.apis.kcp.dev/export` label. This is used in the second storage shadowed by the usual permission claim storage. If an APIExport claims APIBindings, the claim storage will serve the request and with that `claimed.internal.apis.kcp.dev/hash` matters. If the APIExport does not claim it, the other storage will.